### PR TITLE
ncm-metaconfig: add mailrc service

### PR DESCRIPTION
**Pull request by JensTimmerman**
This service allows you to configure mail.rc or .mailrc files

for an example see https://coderwall.com/p/ez1x2w/send-mail-like-a-boss

### CODE REVIEW

File: ncm-metaconfig/src/main/metaconfig/mailrc/tests/regexps/account_config [hunk 1]
```diff
@@ -0,0 +1,9 @@
+account config test for mailrc
+---
+/etc/mail.rc
+multiline
+---
+^account gmail {$
```
Comment (stdweird) [inline ncm-metaconfig/src/main/metaconfig/mailrc/tests/regexps/account_config]: you cannot have unescaped leftbrace here

File: ncm-metaconfig/src/main/metaconfig/mailrc/tests/regexps/account_config [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+^set from=root@gmail.com$
+^set smtp=mailserver.gmail.com$
+^}$
```
Comment (stdweird) [inline ncm-metaconfig/src/main/metaconfig/mailrc/tests/regexps/account_config]: this very likely also requires unescaping

Comment (JensTimmerman) [reply ncm-metaconfig/src/main/metaconfig/mailrc/tests/regexps/account_config]: escaping?

**Comment by jrha** [comment]
Looks good. What is the Dockerfile for?

**Comment by jrha** [comment]
Cool! Any chance you could factor it out into a separate PR? Having it mixed in with the mailrc change is a bit confusing.

**Comment by JensTimmerman** [comment]
Yeah, sorry for not getting back to you, but I'm no longer employed at UGent as a sysadmin, so I'm afraid I won't be following up on quattor development any more in the future.

**Pull request merged**